tests/e2e/utils/auto-efs-setup.py

```diff
@@ -586,6 +592,14 @@ def footer():
     help=f"Specify the path to the source files if different. Default is set to empty.",
     required=False,
 )
+SETUP_FOR_TERRAFORM_DEFAULT = False
+parser.add_argument(
+    "--setup-for-terraform",
```

Comment:
- Can we do `--skip-driver-installation` instead of `--setup-for-terraform`, anyone with driver installed on existing cluster can use this flag
- Also, should we remove `verify_prerequisites`? e.g. eksctl not needed if driver installation is being skipped
- Update the docs reflecting this change

Reply:
done